### DISCUSSION TITLE
Trouble with the Example Data floodcase_202107

### DISCUSSION THREAD

**Discussion by ssomani10** [WAM2layers/WAM2layers#200] [Q&A] [answered]
I decided to create a separate posting after reading some comments from the developer.  I  tried running the floodcase_202107 data and was running with the issue that all data was ending up as 0 in the NETCDF file, while the track moisture and lost moisture is showing us as NAN.  See attached screenshot. No changes were made to the example YAML file and ran as is.

<img width="1214" alt="Backtrack_Error" src="https://user-images.githubusercontent.com/8172915/232643780-951ecff4-eea6-42be-9f02-0b42cc6225b7.png">

Does anyone have any suggestions?  I am thinking the next steps for me is to download the climate data and rerun the analysis to see if there has been any changes.  Thanks.

**Comment by YiyingWang-00** [comment]
I have the same problem，do you have solutions?

**Reply by wzgytuA888** [reply]
Hello, could you please ask if this problem has been resolved

**Comment by ruudvdent** [comment, marked as answer]
We created some updates to the config file in the latest release. Could you check if the problem persists and report back?

**Comment by ssomani10** [comment]
The updated config file definitely works.  I just had to change the directory settings. Thanks.
